common/browser_options.go

```diff
@@ -56,13 +57,14 @@ type LaunchPersistentContextOptions struct {
 }
 
 // NewLaunchOptions returns a new LaunchOptions.
-func NewLaunchOptions(onCloud bool) *LaunchOptions {
+func NewLaunchOptions(onCloud, isRemoteBrowser bool) *LaunchOptions {
```

Comment:
Could create a new function `NewRemoteBrowserLaunchOptions` where it will set `isRemoteBrowser` to `true`.

Reply:
Is another option, personally I prefer to keep a single "generic" constructor to which pass the appropriate flags, but no strong opinion. If we merge both flags and create this new constructor we can get rid of the constants defined in [e24732b](https://github.com/grafana/xk6-browser/pull/800/commits/e24732bd0a56656bc7ea885d305847c83a640f81). But again, I prefer to keep a single constructor.

Reply:
+1 for adding another constructor: Clearer, more readable, and we can eliminate the constants.

Reply:
I prefer a new constructor, but happy to leave it as is.

Reply:
I'm changing this, but the code in `BrowserType` is a little bit "uglier" when calling to `init` method due to the lack of constants, as we have to pass `true` or `false` values. Another alternative is to create the `launchOptions` in `connect` and `launch` methods and then provide that to `init` method so its modified in it. But don't really like it. Will push the first approach and we can discuss.

Reply:
See: [4d5a38f](https://github.com/grafana/xk6-browser/pull/800/commits/4d5a38fdaba893610c859a8d948318401f3cbeb4)

Reply:
If we want to go this way, I suggest moving option parsing to `Connect` and `Launch` and passing the parsed launch options to `init()`. This way:
1. We can eliminate the ugly _local/remote_ bool.
2. More importantly. We'll move Goja stuff to the mapping layer (#271), and I wouldn't bury it in `init()`. Let's leave it at the exported methods while changing this part of the code.